.github/workflows/dispatch-build-bottle.yml

```diff
@@ -50,6 +50,7 @@ jobs:
     runs-on: ubuntu-22.04
     container:
       image: ghcr.io/homebrew/ubuntu22.04:master
+      options: --user=root
```

Comment:
Note that the alternative to this is to `sudo chown "${GITHUB_OUTPUT}"`, but I don't know how to translate that to JavaScript. 🥲

Hopefully someone else can tell us!